wgengine/magicsock/magicsock_windows.go

```diff
@@ -0,0 +1,47 @@
+package magicsock
+
+import (
+	"net"
+
+	"golang.org/x/sys/windows"
+	"tailscale.com/types/logger"
+	"tailscale.com/types/nettype"
+)
+
+// https://github.com/tpn/winsdk-10/blob/9b69fd26ac0c7d0b83d378dba01080e93349c2ed/Include/10.0.16299.0/shared/ws2ipdef.h
+const (
+	IP_MTU_DISCOVER = 71 // IPV6_MTU_DISCOVER has the same value, which is nice.
+)
+
+const (
+	IP_PMTUDISC_NOT_SET = iota
+	IP_PMTUDISC_DO
+	IP_PMTUDISC_DONT
+	IP_PMTUDISC_PROBE
+	IP_PMTUDISC_MAX
```

Comment:
Can you hardcode these values rather than using iota?

Reply:
I wrote it this way because it's an enum in the original C header. But, if you're not a fan I'll just include `IP_PMTUDISC_DO` with its hardcoded value.